dspace/config/registries/schema-publicationVolume-types.xml

```diff
@@ -0,0 +1,36 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>Publication Volume Types</title>
+    </dspace-header>
+
+    <!-- NO EQUIVALENT FOUND:
+        - journalvolume.identifier.name
```

Comment:
Use dc.title?

Reply:
This is indeed configured to use dc.title in this PR: https://github.com/DSpace/DSpace/blob/65c3039c1dfb7d26f0139306785e86aa5ab9a73a/dspace/config/submission-forms.xml#L488